jsapp/js/components/account/securityRoute.tsx

```diff
@@ -139,26 +166,31 @@ export default class SecurityRoute extends React.Component<
 
             <bem.SecurityRow__buttons>
               <ToggleSwitch
-                label={this.state.mfaActive ? t('Enabled') : t('Disabled')}
-                checked={this.state.mfaActive}
+                label={this.state.isMfaActive ? t('Enabled') : t('Disabled')}
+                checked={this.state.isMfaActive}
                 onChange={this.onToggleChange.bind(this)}
               />
             </bem.SecurityRow__buttons>
           </bem.SecurityRow__header>
 
           <bem.SecurityRow__description>
-            {t('Two-factor authentication (2FA) is an added layer of security used when logging into the platform. We recommend enabling Two-factor authentication for an additonal layer of protection.')}
+            {t('Two-factor authentication (2FA) is an added layer of security used when logging into the platform. We recommend enabling Two-factor authentication for an additional layer of protection.')}
```

Comment:
note to self: revise (weird capitalization; other messages use 2FA; standardize this)